starter-project-backend-g32/src/controllers/comment.controller.ts

```diff
@@ -0,0 +1,95 @@
+import {Request , Response } from "express";
+const Comment = require('../models/comment');
+
+const mongoose = require('mongoose');
+const createComment = async (req:Request, res:Response ) => {
+    const comment = new Comment(req.body);
```

Comment:
it's better to get required fields from the body by restructuring it, we may not know what's sent over the body.